src/components/Navbar/index.js

```diff
@@ -31,6 +40,11 @@ const Navbar = ({ toggle }) => {
     <>
       <IconContext.Provider value={{ color: "#fff" }}>
         <Nav scrollNav={scrollNav}>
+          {state.lang === "en" ? (
+            <PlBtn onClick={() => changeLanguage("pl")}></PlBtn>
```

Comment:
you can create one component and extend it with props instead of using two components